Solutions/Marain.Tenancy.OpenApi.Service/Marain/Tenancy/OpenApi/TenancyService.cs

```diff
@@ -411,22 +414,21 @@ public class TenancyService : IOpenApiService
                 throw new OpenApiNotFoundException();
             }
 
-            using (IOperationHolder<RequestTelemetry> opHolder = this.telemetryClient.StartOperation<RequestTelemetry>(DeleteChildTenantOperationId))
+            using IOperationHolder<RequestTelemetry> opHolder = this.telemetryClient.StartOperation<RequestTelemetry>(DeleteChildTenantOperationId);
```

Comment:
If you're editing this, do we still need it? `Corvus.Monitoring` can integrate with Menes to set up operation telemetry automatically for all requests handled by Menes. You just need to ask for it in DI init. Which we might already be doing.